features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml.j2

```diff
@@ -688,6 +688,8 @@
         <RedirectToRequestedRedirectUri>false</RedirectToRequestedRedirectUri>
         {% endif %}
 
+        <PublishPasswordGrantLogin>{{publish.password.grant.auth.login.enable}}</PublishPasswordGrantLogin>
```

Comment:
Any specific reason for dot separators for the words? If there is no hierarchical order for the config then change the dot separators to underscore.

Reply:
Changed.